packages/cli/src/cli.ts

```diff
@@ -6,31 +6,118 @@ import 'reflect-metadata';
 // tslint:disable-next-line: no-import-side-effect
 import './module-name-mapper';
 
+import { CrawlerEntryPoint, setupCrawlerContainer } from 'accessibility-insights-crawler';
 import { isEmpty } from 'lodash';
 import * as yargs from 'yargs';
 import { CliEntryPoint } from './cli-entry-point';
 import { ScanArguments } from './scanner/scan-arguments';
 import { setupCliContainer } from './setup-cli-container';
 
+// tslint:disable-next-line:max-func-body-length
 (async () => {
     const scanArguments = (yargs
-        .usage('Usage: $0 --url <url> --inputFile <inputFile> --output <directoryPath>')
+        .usage(
```

Comment:
I think simulate has no option value and it is optional param, right?
maxUrl may be rename to maxUrlCount or similar
Include each param in `[]` if optional